docs/locale/fr/LC_MESSAGES/userguide/installation.po

```diff
@@ -7,192 +7,246 @@ msgid ""
+"identifiés` dans `Confidentialité et sécurité Système > Confidentialité et "
+"sécurité` :"
 
 #: ../../userguide/installation.rst:-1
-msgid "MacOS path to Security and Privacy"
-msgstr "Chemin vers Sécurité et Confidentialité dans MacOS"
+msgid "macOS path to Security and Privacy"
+msgstr "macOS chemin vers Sécurité et Confidentialité"
 
 #: ../../userguide/installation.rst:-1
-msgid "MacOS previous allow identified developer"
-msgstr "MacOS ancien autoriser développeurs identifiés"
+msgid "macOS previous allow identified developer"
+msgstr "macOS ancien autoriser développeurs identifiés"
 
-#: ../../userguide/installation.rst:87
+#: ../../userguide/installation.rst:113
 msgid ""
 "To change this setting, click the lock first which will require admin rights."
 msgstr ""
 "Pour changer ce paramètre, cliquez d'abord sur le cadenas, qui requiert les "
 "droits d'administrateur."
 
-#: ../../userguide/installation.rst:89
+#: ../../userguide/installation.rst:115
 msgid ""
-"Once the installation is done, a `System Extension Updated` window will pop "
-"up. Click `Open Security Preferences`, and click the lock, then click "
-"`Allow`:"
+"Once the installation is done, a `System Extension Updated` window will pop up. "
+"Click `Open Security Preferences`, and click the lock, then click `Allow`:"
 msgstr ""
 "Une fois l'installation terminée, une fenêtre suggérant d'autoriser cette "
-"extension système apparaît. Cliquez sur `Ouvrir les préférences de "
-"sécurité`, puis cliquez sur `Autoriser` après avoir cliqué à nouveau sur le "
-"cadenas :"
+"extension système apparaît. Cliquez sur `Ouvrir les préférences de sécurité`, "
+"puis cliquez sur `Autoriser` après avoir cliqué à nouveau sur le cadenas :"
 
 #: ../../userguide/installation.rst:-1
 msgid "Previous System Extension Updated window"
 msgstr "Ancienne fenêtre de mise à jour d'extension système"
 
 #: ../../userguide/installation.rst:-1
-msgid "MacOS previous allow extension"
-msgstr "Ancien autoriser extension MacOS"
+msgid "macOS previous allow extension"
+msgstr "macOS ancien autoriser extension"
 
-#: ../../userguide/installation.rst:99
-msgid "Finally, reboot your Mac to complete the installation."
-msgstr "Enfin, redémarrez votre Mac pour finaliser l'installation."
+#: ../../userguide/installation.rst:129
+msgid "Install an older version of Parsec"
+msgstr "Installer une ancienne version de Parsec"
+
+#: ../../userguide/installation.rst:133
+msgid ""
+"For security reasons, you should **always install the latest stable version of "
+"Parsec** as it contains the latest security fixes."
+msgstr ""
+"Pour des raisons de sécurité, vous devriez **toujours installer la dernière "
+"version stable de Parsec** car elle contient les derniers correctifs de sécurité."
+
+#: ../../userguide/installation.rst:135
+msgid "If you need to access an older version, they are available on `GitHub`_."
+msgstr ""
+"Si vous avez besoin d'accéder à une version plus ancienne, elle est disponible "
```

Comment:
**wording**: 

```suggestion
msgid "If you need to access older versions, they are available on `GitHub`_."
msgstr ""
"Si vous avez besoin d'accéder à des versions plus ancienne, elles sont disponible "
```

The `they are` mean we talk about multiple older versions, so I propose to use the plural form on the first part of the phrase.

But we could do the inverse and use a singular form on the second part :thinking: 

With that change we are more in-phase between french & english.